warehouse/templates/email/added-as-collaborator.body.txt

```diff
@@ -1,6 +1,17 @@
-You have been added as {{ role }} to the PyPI project {{ project }} by {{ submitter }}.
+You have been added as a collaborator to the PyPI project {{ project }}
```

Comment:
We should lose the reference to `PyPI` here, since this same email could be sent for TestPyPI.